start-user

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I don't see where this file gets used

Reply:
Somehow I missed removing this file. It's nothing but remains from another solution I had.